src/pt/maidscan/build.gradle

```
@@ -2,7 +2,7 @@ ext {
     extName = 'Maid Scan'
     extClass = '.MaidScan'
     themePkg = 'greenshit'
-    baseUrl = 'https://novo.empreguetes.site'
+    baseUrl = 'https://empreguetes.xyz/'
```

Remove the slash at the end of the url
```suggestion
    baseUrl = 'https://empreguetes.xyz'
```